_component_design/switches.md

```diff
@@ -14,72 +14,91 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-<!--{% include todo.html content="Update focus states, expand on inline validation, and improve
-keyboard accessibility." %}-->
 
-The switch should only be used in **mobile** designs. Enabling preferences or
-settings is a prime use case for this component. The switch can be used with or
-without text, depending on the context provided in the switch label. If more
-complex text is needed in the switch than yes/no or on/off, use a **toggle**.
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-## Default
+### When to use
+{: .hxSubSectionTitle}
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/selectors-switch.png){:width="30%"}
+Use a switch if the user is turning something on or off, or saying yes or no to a question. There is no neutral option; the answer is either set to on or off. Enabling preferences or settings is a prime use case for this component. The switch should always be used with text. 
+
+If more complex text is needed in the switch than yes/no or on/off, use a toggle. A switch should NOT be used for things like user agreements or if there are more than two options available.
+
+
+### Best Practice
+{: .hxSubSectionTitle}
+- Switches are only used for on/off and yes/no, and are usually only used for account and user settings, billing, and notifications. If your use case is outside of these three areas, consider a radio button or checkbox. 
+- Changing the state of the switch should be saved immediately. Use a checkbox or radio button if the user has to perform extra steps for changes to be effective (for example, if the user must click a "submit" or "next" button to apply changes). Use check boxes when the user can select multiple items that are related to a single setting or feature.
```

Comment:
Save any changes to the state of the switch immediately.*

Reply:
check-box and check-boxes*

Reply:
"Use check-boxes when the user requires the selection of multiple items that are related to a single setting or feature."

Reply:
Updated.